back/engines/commercial/flag_inappropriate_content/spec/acceptance/initiatives_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
```

Comment:
Calling @kogre here just to make sure but I think you might want to either make sure these are `doc: false` tests, or these might need to be moved to the core specs with `CitizenLab.ee?` checks.

Reply:
I think in this case, `doc: false` makes a lot of sense because it's useless information, which is better hidden, for users of the API.

But then, if we do so, I also think it's better to remove the `parameter` etc definitions, as they're only useful for documentation purposes and we don't want to have to specify and maintain them both in this file and in the initiatives_spec in core.

So if we do that, to me it makes sense to keep them here then, as then they're really only about flag_inappropriate_content`.

Reply:
The `parameter` definitions cannot be removed without breaking the specs, so I kept them but removed the strings that document the parameters.